webpack.config.mjs

```diff
@@ -60,6 +82,14 @@ const config = {
 					},
 				],
 			},
+			{
+				test: /.js$/,
+				include: /prebid\.js/,
+				use: {
+					loader: 'babel-loader',
+					options: prebidBabelOptions,
```

Comment:
prebid has some odd babel configuration so we need to use their babel config, this is recommended in the prebid docs.